### DISCUSSION TITLE
Radio Button UserData Propety

### DISCUSSION THREAD

**Discussion by Zach-612** [gluonhq/scenebuilder#609] [Ideas]
The easiest way I've found to get an identifier for the active radio button in a specific toggle group is by getting its userData. 

> _ToggleGroup_.getSelectedToggle().getUserData().toString()

But there doesn't seem to be any way to add a userData property through SceneBuilder.  I end up going through the .fxml file and adding it to each radio button line manually.  It would be nice to have a "User Data" field in the radio button properties menu.

**Comment by Oliver-Loeffler** [comment]
This sounds like a really good idea. This is worth to be explored as it would add some useful functionality and comfort to Scene Builder.
